hack/hive/hive-dev-install.sh

```diff
@@ -4,49 +4,39 @@ set -o errexit \
     -o nounset
 
 declare -r utils=hack/util.sh
-if [ -f "$utils" ]; then
-    # shellcheck source=../util.sh  
-    source "$utils"  
+if [[ -f "$utils" ]]; then
+    # shellcheck source=../util.sh
+    source "$utils"
 fi
 
 HIVE_OPERATOR_NS="hive"
-KUBECTL="$( which kubectl 2> /dev/null || which oc 2> /dev/null)"
+KUBECTL="$(which kubectl 2> /dev/null || which oc 2> /dev/null)"
```

Comment:
ik that this is a line of code that is effectively "unchanged" - but `oc` requires the installation of a specific pinned version of `kubectl` so this is a weird command :sweat_smile: